packages/account-management/src/frontend/AccountManagement/DeleteAccount.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { SolidButton } from '@weco/common/views/components/ButtonSolid/ButtonSolid';
+import SpacingComponent from '@weco/common/views/components/SpacingComponent/SpacingComponent';
+
+const logo = 'https://identity-public-assets-stage.s3.eu-west-1.amazonaws.com/images/wellcomecollections-150x50.png';
```

Comment:
Looks like this URL is just to the staging environment, which isn't a huge deal right now but we'll need to change before we deploy to prod. It's in a few other places too. Can we either stick a `TODO` before each usage, or get them all from a common place (e.g. in `config.ts`) with a `TODO` on that?